test/worker_test.py

```diff
@@ -1322,6 +1322,17 @@ class NotImportedTask(luigi.Task):
             self.assertTrue(self.assistant.run())
             self.assertEqual(list(self.sch.task_list('DONE', '').keys()), [task.task_id])
 
+    def test_unimported_job_sends_failure_message(self):
+        class NotInAssistantTask(luigi.Task):
```

Comment:
That's only used when you want the module to exist. I'm only testing what happens when it doesn't exist.